.travis.yml

```diff
@@ -11,6 +11,8 @@ env:
       - CLEAN_TRAVIS_TAG=${TRAVIS_TAG/[[:space:]]/}
       - COMMIT=${CLEAN_TRAVIS_TAG:-${TRAVIS_COMMIT:0:7}}
 
+if: (type = pull_request) OR (tag IS present) OR (branch = develop) OR (branch =~ /^feature.*/)
```

Comment:
Does this fix https://github.com/azavea/raster-vision/issues/268?

Reply:
Yep! I'll mark it in the PR description.